SDBlockDevice.cpp

```diff
@@ -147,6 +147,11 @@
 #include "mbed_debug.h"
 #include <errno.h>
 
+/* Required version: 5.5.4 and above */
+#if (MBED_VERSION < MBED_ENCODE_VERSION(5,5,4))
```

Comment:
This is a bit ugly. Could we have a macro or something similar for application / library use? Would be good to prototype here and move into mbed OS where the version comes from.
```
#define MBED_VERSION_CHECK(5,5,4)
```